fontforge/parsettfatt.c

```diff
@@ -776,6 +776,14 @@ static AnchorClass **MarkGlyphsProcessMarks(FILE *ttf,int markoffset,
     struct mr { uint16 class, offset; } *at_offsets;
     SplineChar *sc;
 
+    fseek(ttf,markoffset,SEEK_SET);
```

Comment:
Ok yep that makes sense. My only concern was if `ttf` (the file pointer) was being used in the code block that you moved it around, but on checking it isn't.